test/test_plugin.rb

```diff
@@ -0,0 +1,40 @@
+require_relative "helper"
+require_relative "helpers/integration"
+
+class TestPlugin < TestIntegration
+  def test_plugin
+    skip "Skipped on Windows & Ruby 2.4.x" if windows? && RUBY_VERSION < '2.5.0'
```

Comment:
Should leave a comment here as to why

Reply:
IDC.  It's Windows specific, and 2.4 is 'Currently in security maintenance phase.'  On Windows, I suspect there are very few people using 2.4.

Reply:
Fixed.